react/scss/person-graph.scss

```diff
@@ -0,0 +1,19 @@
+@import 'common';
+
+.link line {
+  stroke: #999;
+  stroke-opacity: .6;
+}
+
+.node--group {
+  cursor: pointer;
+}
+
+.node--leaf {
```

Comment:
same for label--leaf, also the convention when you style modifier is to nest them in the parent:

``` 
node {
   &-- leaf { 
     // ...
  }
}
```